docs/getting-started.md

```diff
@@ -8,14 +8,42 @@ make tutorial-image  # create the Docker image for the tutorial
 make tutorial-run
 ```
 
-Once the Ghidra server has started and the Jupyter notebook is up (this should take about one minute), follow the displayed instructions to access the Jupyter noteboks and have fun!
+It takes a minute for the notebook to start up. The jupyter notebook URL and token will finally print on the terminal.
+Once running, you can access the tutorial from [localhost:8888](http://localhost:8888) with your web browser. Have fun!
+
+## GUI
+OFRAK comes with a web-based GUI frontend for visualizing and manipulating binary targets. To activate the GUI from the
+OFRAK tutorial container, run:
+
+```
+docker exec -it ofrak-tutorial \
+  bash -c "nginx; python /ofrak_server.py 0.0.0.0 8877"
+```
+
+Once running, access [localhost:80](http://localhost:80) and start by dropping anything you'd like into it!
+
+Note that for now, the frontend only supports the `ofrak-tutorial.yml` and `ofrak-ghidra.yml` image build targets.
```

Comment:
Nope, it runs in all except the tutorial and angr ones as the default entrypoint!

```
$ grep -i frontend *.yml
ofrak-tutorial.yml:    "frontend",
ofrak-ghidra.yml:    "frontend",
ofrak-dev.yml:    "frontend",
ofrak-binary-ninja.yml:    "frontend",
ofrak-core-dev.yml:    "frontend",
ofrak-angr.yml:    "frontend",
```

Here is an excerpted example of one of the `base.Dockerfile` files that is generated.

``` dockerfile
FROM node:latest AS svelte
COPY --chown=node:node frontend /home/node/frontend
WORKDIR /home/node/frontend
RUN su node -c "npm install && npm run build"

# syntax = docker/dockerfile:1.3
FROM python:3.7-bullseye@sha256:338ead05c1a0aa8bd8fcba8e4dbbe2afd0283b4732fd30cf9b3bfcfcbc4affab

# [ ... ]

RUN apt-get update && apt-get install --yes nginx
RUN python3 -m pip install --upgrade aiohttp

ENTRYPOINT nginx \
  & python3 -m ofrak_ghidra.server start \
  & python3 /ofrak_server.py 0.0.0.0 8877 ghidra
```